usedHttp.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
+
+void makeRequest() async{
+    var response = await http.get('https://example.com/');
```

Comment:
Can you use `final` instead of `var`, since you don't change the value of `response`?

In general, prefer using `final` instead of `var` to avoid bugs.

Reply:
I will now use `final` instead of `var` if the value doesn't change.